test/run

```diff
@@ -429,43 +398,34 @@ function run_dockerfiles_test() {
   rm "$dockerfile_tmp"
 }
 
+TEST_SUMMARY=''
+ct_enable_cleanup
+
 CID_FILE_DIR=$(mktemp -d)
 
 build_image "test-app"
 
 # Run the chosen tests
-TEST_LIST=${TEST_LIST_APP} run_all_tests "test-app"
+TEST_SET=${TEST_LIST_APP} ct_run_tests_from_testset "test-app"
 
 build_image "start-hook-test-app"
 
 # Run the chosen tests
-TEST_LIST=${TEST_LIST_HOOK_APP} run_all_tests "start-hook-test-app"
+TEST_SET=${TEST_LIST_HOOK_APP} ct_run_tests_from_testset "start-hook-test-app"
 
 build_image "perl-test-app"
 
 # Run the chosen tests
-TEST_LIST=${TEST_LIST_PERL_APP} run_all_tests "perl-test-app"
+TEST_SET=${TEST_LIST_PERL_APP} ct_run_tests_from_testset "perl-test-app"
 
 # Run the rest of the tests by defining TEST_LIST to include the rest of the relevant cases
-TEST_LIST=run_dockerfiles_test
+TEST_SET=run_dockerfiles_test
 
 # These tests only make sense for rhscl
 if [ "$OS" == "rhel7" ] || [ "$OS" == "centos7" ]; then
-  # autocleanup only enabled here as only the following tests so far use it
-  ct_enable_cleanup
```

Comment:
Why this was removed?

Reply:
Because it is added outside of this `if`.